src/test/java/io/knotx/launcher/KnotxStarterVerticleTest.java

```diff
@@ -114,6 +114,32 @@ void startModuleWithSystemPropertyValue(VertxTestContext testContext, Vertx vert
                 .setTestContext(testContext))
     );
 
+    // when
+    vertx.rxDeployVerticle(KnotxStarterVerticle.class.getName(), options)
+        .subscribe(
+            success -> testContext.completeNow(),
+            testContext::failNow
+        );
+  }
+  @Test
+  @DisplayName("Deploy a module with a configuration included from a separate file and property defined in system properties.")
+  void startModuleWithIncludesWithSystemPropertyValue(VertxTestContext testContext, Vertx vertx) {
```

Comment:
I think that this test combines two others:
```
io.knotx.launcher.KnotxStarterVerticleTest#startModuleWithIncludes
io.knotx.launcher.KnotxStarterVerticleTest#startModuleWithSystemPropertyValue
```
Not sure that we need this combination in tests.